File: sample/src/main/java/me/xiaopan/assemblyadaptersample/MainActivity.java
package me.xiaopan.assemblyadaptersample;

import android.os.Build;
import android.os.Bundle;
import android.support.v4.app.Fragment;
import android.support.v4.app.FragmentActivity;
import android.support.v4.view.ViewPager;

import me.xiaopan.assemblyadapter.FragmentArrayPagerAdapter;
import me.xiaopan.assemblyadaptersample.fragment.ExpandableListViewFragment;
import me.xiaopan.assemblyadaptersample.fragment.GridRecyclerViewFragment;
import me.xiaopan.assemblyadaptersample.fragment.ListViewFragment;
import me.xiaopan.assemblyadaptersample.fragment.PagerAdapterFragment;
import me.xiaopan.assemblyadaptersample.fragment.RecyclerViewFragment;
import me.xiaopan.assemblyadaptersample.fragment.SpinnerFragment;
import me.xiaopan.assemblyadaptersample.fragment.ViewPagerFragment;
import me.xiaopan.psts.PagerSlidingTabStrip;

public class MainActivity extends FragmentActivity {

    @Override
    protected void onCreate(Bundle savedInstanceState) {
        super.onCreate(savedInstanceState);

        setContentView(R.layout.activity_main);

        ViewPager viewPager = (ViewPager) findViewById(R.id.pager_mainActivity_content);
        viewPager.setAdapter(new FragmentArrayPagerAdapter(getSupportFragmentManager(), new Fragment[]{
                new ListViewFragment(),
                new RecyclerViewFragment(),
                new GridRecyclerViewFragment(),
                new ExpandableListViewFragment(),
                new SpinnerFragment(),
                new ViewPagerFragment(),
                new PagerAdapterFragment(),
        }));

        PagerSlidingTabStrip tabStrip = (PagerSlidingTabStrip) findViewById(R.id.tabStrip_mainActivity_tabs);
        tabStrip.setViewPager(viewPager);

        if(Build.VERSION.SDK_INT >= Build.VERSION_CODES.LOLLIPOP){
            getActionBar().setElevation(0);
        }
    }
}

File: sample/src/main/java/me/xiaopan/assemblyadaptersample/MainActivity.kt
package me.xiaopan.assemblyadaptersample

import android.os.Build
import android.os.Bundle
import android.support.v4.app.FragmentActivity
import android.support.v4.view.ViewPager
import me.xiaopan.assemblyadapter.FragmentArrayPagerAdapter
import me.xiaopan.assemblyadaptersample.fragment.*
import me.xiaopan.psts.PagerSlidingTabStrip

class MainActivity : FragmentActivity() {

    override fun onCreate(savedInstanceState: Bundle?) {
        super.onCreate(savedInstanceState)

        setContentView(R.layout.activity_main)

        val viewPager = findViewById(R.id.pager_mainActivity_content) as ViewPager
        viewPager.adapter = FragmentArrayPagerAdapter(supportFragmentManager, arrayOf(
                ListViewFragment(),
                RecyclerViewFragment(),
                GridRecyclerViewFragment(),
                ExpandableListViewFragment(),
                SpinnerFragment(),
                ViewPagerFragment(),
                PagerAdapterFragment()))

        val tabStrip = findViewById(R.id.tabStrip_mainActivity_tabs) as PagerSlidingTabStrip
        tabStrip.setViewPager(viewPager)

        if (Build.VERSION.SDK_INT >= Build.VERSION_CODES.LOLLIPOP) {
            actionBar!!.elevation = 0f
        }
    }
}